server/controllers/users.js

```diff
@@ -150,6 +151,41 @@ class UserController {
       next(error);
     }
   }
+
+  /**
+  * Get users and their corresponding profiles
+  * @async
+  * @param {object} req - Request object
+  * @param {object} res - Response object
+  * @param {object} next The next middleware
+  * @return {json} Returns json object
+  * @static
+  */
+  static async getUsers(req, res, next) {
+    try {
+      const users = await User.findAll({
+        include: [
+          {
+            model: models.Profile,
+            as: 'profile'
+          }
+        ]
+      });
+
+      if (isEmpty(users)) {
+        return Response.error(res, 400, 'No user found');
+      }
+
+      return res.status(200)
+        .send({
+          status: 'success',
+          message: 'Users and corresponding profiles retrieved successfully',
+          users
```

Comment:
So you would basically allow all users see the hashed passwords of all other users (and some other sensitive details) ?

Reply:
Before now I did not know how to perform exclude queries in sequelize. So I studied the querying section in their docs and found how to exclude columns in query attributes. I implemented it and was able to exclude sensitive information.

Reply:
also, return users as `payload`

```javascript
...
payload: users
});
```